lib/config/hadeployconfigverify.go

```diff
@@ -14,6 +19,12 @@ func (c *HaDeployConfig) Verify() error {
 
 	// Validate ExistingInfra (on prem)
 	if c.IsExistingInfra() {
+		if c.ObjectStorage != nil && c.ObjectStorage.Config.Location == GCS_STORAGE {
+			err := c.readGCSConfigFile(c.ObjectStorage.Config)
```

Comment:
We are reading the GcsServiceAccount.json here then do we need the changes which made in Parse function (lib/config/hadeployconfig.go)? Here also we are reading the file. and then invoking the same Verify function and now in verify function we are handling it.
Same for Post Deployment: https://github.com/chef/automate/pull/8176 we have to remove all these changes right?